docker/Dockerfile

```diff
@@ -6,9 +6,17 @@ RUN go get golang.org/x/tools/cmd/vet
 RUN go get github.com/golang/lint/golint
 RUN go get github.com/kisielk/errcheck
 RUN go get github.com/tools/godep
+RUN curl -sL -o /usr/bin/gimme https://raw.githubusercontent.com/travis-ci/gimme/master/gimme
+RUN chmod +x /usr/bin/gimme
+ADD ./test.sh /opt/test.sh
```

Comment:
Please use `COPY`, it's better than `ADD` in this context (see [this](https://docs.docker.com/articles/dockerfile_best-practices/#add-or-copy))